feature_list/feature_list.cpp

```diff
@@ -267,7 +267,7 @@ int main(int argc, char ** argv)
     for (size_t ftIndex = 0; ftIndex < loader.GetNumFeatures(); ftIndex++)
     {
       FeatureType ft;
-      if (loader.GetFeatureByIndex(ftIndex, ft))
+      if (loader.GetFeatureByIndex(static_cast<uint32_t>(ftIndex), ft))
```

Comment:
So, in this case it's better to modify GetNumFeatures(), right? As you can read only by uint32_t, it's reasonable that the number of elements is uint32_t also.